spec/mspec/.travis.yml

```diff
@@ -23,8 +20,10 @@ matrix:
       # https://github.com/travis-ci/travis-ci/issues/8978
       - gem update --system
       - gem install bundler
-  - jdk: oraclejdk8
+  - rvm: system
     install:
-      - curl -L https://github.com/oracle/truffleruby/releases/download/vm-enterprise-0.28/truffleruby-testing-0.28.tar.gz | tar xz
-      - source truffleruby/setup_env
+      - curl -L https://github.com/oracle/truffleruby/releases/download/vm-1.0.0-rc2/truffleruby-1.0.0-rc2-linux-amd64.tar.gz | tar xz
```

Comment:
@eregon mistake right?

Reply:
It's just the [.travis.yml](https://github.com/ruby/mspec/blob/master/.travis.yml) of ruby/mspec, the changes there can be ignored.

Reply:
Maybe I should refine the [import script](https://github.com/ruby/mspec/blob/master/tool/pull-latest-mspec-spec) to not import .travis.yml.

Reply:
Done: https://github.com/ruby/mspec/commit/b3dbbed2843fd3ad422d990ad5febf5849b47c9f
It will be removed on the next specs update.

Reply:
thanks!